components/Label/Label.tsx

```diff
@@ -34,3 +34,4 @@ const Label = React.forwardRef<
 Label.displayName = LabelPrimitive.Root.displayName;
 
 export { Label };
+
```

Comment:
NIT: How come there's an extra line here?

Reply:
... and now it's gone!